repos/c2sm/packages/py-gt4py/package.py

```diff
@@ -40,12 +40,12 @@ class PyGt4py(PythonPackage):
     depends_on('py-deepdiff@5.6:', type=('build', 'run'))
     depends_on('py-devtools@0.6:', type=('build', 'run'))
     depends_on('py-frozendict@2.3:', type=('build', 'run'))
-    depends_on('py-gridtools-cpp@2.2.3:', type=('build', 'run'))
+    depends_on('py-gridtools-cpp@2.3.0', type=('build', 'run'))
```

Comment:
Could you make the version number of py-gridtools a minimum requirement?
Otherwise each new version requires a change of the icon-recipe.

Reply:
done!